src/javascripts/app.js

```diff
@@ -0,0 +1,11 @@
+import LoginController from "./controllers/loginController";
+import AccountService from "./services/accountService";
+import LoginView from "./views/loginView";
+
+    document.addEventListener("DOMContentLoaded", () => {
+      const loginController = new LoginController(new AccountService(), new LoginView());
+      loginController.init();
+      console.log('app.js')
```

Comment:
Remove this

Reply:
@phuocduong-agilityio Do you mean delete `console.log `or do you mean delete it all. Sorry I don't understand what you mean